kernel/packages/shared/apis/SceneStateStorageController/BuilderServerAPIManager.ts

```diff
@@ -155,55 +156,92 @@ export class BuilderServerAPIManager {
     return builderManifest
   }
 
-  async createManifestFromSerializedState(
+  async builderManifesFromSerializedState(
     builderSceneId: string,
     builderProjectId: string,
     baseParcel: string,
     parcels: string[],
     title: string | undefined,
     description: string | undefined,
+    ethAddress: string,
     scene: SerializedSceneState,
-    identity: ExplorerIdentity
-  ): Promise<BuilderManifest | undefined> {
-    try {
```

Comment:
Why are we removing this try/catch? Won't this make the client crash if something fails in builder in world or something like that?

Reply:
Actually the whole function was deleted. The diff here is misleading and looks like just a couple of things where removed :P
We can add a try/catch again just to be safe, but I don't think is necessary. Also the function `builderManifestFromSerializedState` is called inside a try/catch block